core/federated/RTI/CMakeLists.txt

```diff
@@ -76,12 +77,22 @@ add_executable(
     ${CoreLib}/utils/pqueue_tag.c
     ${CoreLib}/utils/pqueue.c
 )
+if (NOT DEFINED LOG_LEVEL)
+    set(LOG_LEVEL 0)
+ENDIF(NOT DEFINED LOG_LEVEL)
 
 IF(CMAKE_BUILD_TYPE MATCHES DEBUG)
     # Set the LOG_LEVEL to 4 to get DEBUG messages
     message("-- Building RTI with DEBUG messages enabled")
-    target_compile_definitions(RTI PUBLIC LOG_LEVEL=4)
+    set(LOG_LEVEL 4)
 ENDIF(CMAKE_BUILD_TYPE MATCHES DEBUG)
+target_compile_definitions(RTI PUBLIC LOG_LEVEL=${LOG_LEVEL})
+
+include(${CMAKE_CURRENT_LIST_DIR}/../../../plugin-defaults/trace/CMakeLists.txt)
```

Comment:
This include is an artifact of the fact that the RTI is treated as a separate cmake project, which it probably shouldn't. This is something we can resolve in a later PR.